### PR TITLE
:scroll: Update ChangeLog for new version. [skip ci]

### PR DESCRIPTION
**Pull request by kaikai1024**

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -4,101 +4,46 @@ All notable changes to this project will be documented in this file. And this pr
...
 ### Network
 
-- [Feature] Enable parsing hostname directly in network.toml. [@driftluo]
-- [Fix] Fix bug for network not send all msg. [@jerry-yu]
-
 ### Consensus
 
-- [Optimization] Add a min-heap timer. [@KaoImin]
-- [Optimization] Optimize wait time for proposal, prevote and precommit. [@jerry-yu]
+- [Optimization] Use crossbeam channel. [@kaikai]
```
Comment (classicalliu) [inline ChangeLog.md]: May "Replace std::channel with crossbeam channel." be better?

Comment (kaikai1024) [reply ChangeLog.md]: > May "Replace std::channel with crossbeam channel." be better?

yeah.

Comment (kaikai1024) [reply ChangeLog.md]: > May "Replace std::channel with crossbeam channel." be better?

done. Actuall it's `std::sync::mpsc::channel;`

**Comment by u2** [comment]
Why do you remove history?

**Comment by kaikai1024** [comment]
> Why do you remove history?

My fault. Please check the pr #268 and issue #269

**Pull request merged**